lib/oss/version.rb

```diff
@@ -0,0 +1,3 @@
+module SlideHub
+  VERSION = '1.0.0'
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.